releasenotes.md

```diff
@@ -6,6 +6,18 @@
 
 # Known Issues
 
-* NA
+* **Name or service not known** 
+
+If you encounter an error such as this during execution on ParallelCluster:
+
+```
+queue-trn2-dy-compute-resource-trn2-1:10363:10409 [0] include/socket.h:235 NCCL WARN Net : error encountered when getting address info : Name or service not known
```

Comment:
Can you add this as well to the error message?
NCCL WARN Invalid NCCL_COMM_ID [queue-trn2-dy-compute-resource-trn2-1:10363:10409] please use format: <ipv4>:<port> or [<ipv6>]:<port> or <hostname>:<port>

Reply:
ok let me do it

Reply:
Added